src/Utils/Helpers/formatToCurrency.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * @author Idan Gigi idangi@matrix.co.il
+ * @param {string} amount
+ * @returns {string} 1000 => 1,000.00, 1,000,000.123 => 1,000,000.12
+ */
+export const formatToCurrency = (amount) => {
```

Comment:
I think is better to put this file in clinikal-react/src/Utils/Helpers/validation